<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupEarthCoverage</class>
 <widget class="QWizard" name="SetupEarthCoverage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Define earth coverage areas and lines</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::HelpButtonOnRight</set>
  </property>
  <widget class="QWizardPage" name="wizardPage_2">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>1- Choose the frame layer</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="filterLineEdit"/>
      </item>
      <item>
       <widget class="QListView" name="listView"/>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>488</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>2- Choose the area classes that build earth coverage (only parent classes are listed here).</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="CustomSelector" name="areasCustomSelector" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>3- Choose the line classes that, along with the frame, build the areas of earth coverage (only parent classes are listed here).
If there is a constraint attribute in a selected class, right click it and define it.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="CustomSelector" name="linesCustomSelector" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>675</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3- For each area class of earth coverage, define which lines are used to build it (note that frame is always used to close areas)</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>801</width>
      <height>381</height>
     </rect>
    </property>
    <property name="itemsExpandable">
     <bool>true</bool>
    </property>
    <attribute name="headerCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="headerMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="headerStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Class</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Delimiters</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomSelector</class>
   <extends>QWidget</extends>
   <header>DsgTools.CustomWidgets.customSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
